pydrad/parse/parse.py

```diff
@@ -336,168 +341,74 @@ def __repr__(self):
 Filename: {self._phy_filename}
 Timestep #: {self._index}"""
 
-    def _read_phy(self):
+    def _read_amr(self):
         """
-        Parse the hydrodynamics results file
+        Parse the adaptive mesh grid file
         """
-        self._phy_data = np.loadtxt(self._phy_filename)
+        self._amr_data = read_amr_file(self._amr_filename)
 
-    def _read_amr(self):
+    def _read_phy(self):
         """
-        Parse the adaptive mesh grid file
+        Parse the hydrodynamics results file
```

Comment:
```suggestion
        Parse the physical variables (.PHY) file
```

Nitpicking just to clarify what the file extension means